modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts

```diff
@@ -62,7 +62,7 @@ export async function parseI3STileContent(
 
   if (tileOptions.textureUrl) {
     // @ts-expect-error options is not properly typed
-    const url = getUrlWithToken(tileOptions.textureUrl, options?.i3s?.token);
+    const url = getUrlWithToken(getInternalUrl(tileOptions.textureUrl), options?.i3s?.token);
```

Comment:
Is there a better (more self documenting) name than `getInternalUrl`? Not quite clear what that does from just seeing it like this.